### PR TITLE
Update rejection reasons

### PR DESCRIPTION
**Pull request by henryhadlow**
Update wording as requested by Parliament
Re-order based on data

### CODE REVIEW

File: spec/helpers/rejection_helper_spec.rb [hunk 1]
```diff
@@ -11,7 +11,8 @@
 
   describe "#rejection_description" do
     it "gives rejection description from the locale file" do
-      expect(helper.rejection_description(code)).to eq('<p>There’s already a petition about this issue.</p>')
+      expect(helper.rejection_description(code)).to eq('<p>There’s already a petition about this issue. We cannot accept a new petition when we already have one about a very similar issue.</p>
+<p>You are more likely to get action on this issue if you sign and share a single petition.</p>')
```
Comment (pixeltrix) [inline spec/helpers/rejection_helper_spec.rb]: I wonder whether changing this assertion to use a `match` on a partial string would be more readable, e.g:

``` ruby
expect(helper.rejection_description(code)).to match(%r[<p>There’s already a petition about this issue</p>])
```

Just nit-picking - I'll leave it up to you :smile:

Comment (henryhadlow) [inline spec/helpers/rejection_helper_spec.rb]: Good idea! Coming up...

On 21 September 2015 at 14:20, Andrew White <notifications@github.com>
wrote:

> In spec/helpers/rejection_helper_spec.rb
> <https://github.com/alphagov/e-petitions/pull/394#discussion_r39968721>:
>
> > @@ -11,7 +11,8 @@
> >
> >    describe "#rejection_description" do
> >      it "gives rejection description from the locale file" do
> > -      expect(helper.rejection_description(code)).to eq('<p>There’s already a petition about this issue.</p>')
> > +      expect(helper.rejection_description(code)).to eq('<p>There’s already a petition about this issue. We cannot accept a new petition when we already have one about a very similar issue.</p>
> > +<p>You are more likely to get action on this issue if you sign and share a single petition.</p>')
>
> I wonder whether changing this assertion to use a match on a partial
> string would be more readable, e.g:
>
> expect(helper.rejection_description(code)).to match(%r[<p>There’s already a petition about this issue</p>])
>
> Just nit-picking - I'll leave it up to you [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/e-petitions/pull/394/files#r39968721>.
>



-- 
Henry Hadlow
Designer
Government Digital Service <https://gds.blog.gov.uk/>

**Comment by pixeltrix** [comment]
@henryhadlow I see you've done some capitalization changes regarding Government/government in this PR - can you respond to #379 on which form we should be using, thanks.

**Comment by henryhadlow** [comment]
Good point. I'll ask the content team.

On 21 September 2015 at 14:23, Andrew White <notifications@github.com>
wrote:

> @henryhadlow <https://github.com/henryhadlow> I see you've done some
> capitalization changes regarding Government/government in this PR - can you
> respond to #379 <https://github.com/alphagov/e-petitions/pull/379> on
> which form we should be using, thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/e-petitions/pull/394#issuecomment-141974343>.
>



-- 
Henry Hadlow
Designer
Government Digital Service <https://gds.blog.gov.uk/>

**Pull request merged**